### DISCUSSION TITLE
What's the design system behind the UI?

### DISCUSSION THREAD

**Discussion by gotzmann** [pocketbase/pocketbase#2664] [Q&A] [answered]
UI looks cool, I'm wonder - are there some component / design library behind it? If I'd like to use similar looking components and styles - are there some prototypes elsewhere or they were built for PocketBase from the scratch?

**Comment by ganigeorgiev** [comment]
The UI was built from scratch. There are plans sometime in the future to refactor and export the components as part of https://github.com/pocketbase/pocketbase/issues/898, but for now remains a low priority.

**Comment by gotzmann** [comment]
Gani I'm a fan of your coding and styling genius :) While I'm wrapping my head into how all the gears of Svelte are moving here, please let me know if it possible to hire you for some similar work outside of your projects? Feel free to DM me: sgotsulyak@gmail.com

**Reply by ganigeorgiev** [reply, marked as answer]
> I'm wrapping my head into how all the gears of Svelte are moving here

The UI code could be found in https://github.com/pocketbase/pocketbase/tree/master/ui but while you can use it as a reference, keep in mind that it is a little messy and I wouldn't recommend it as an example for a good structured Svelte project. At the moment it is a implemented as nonextendable/internal part of the whole application to allow faster iteration since some things are not clearly defined yet.

> please let me know if it possible to hire you for some similar work outside of your projects?

Thank you, but at least until I manage to release [Presentator v3](https://github.com/presentator/presentator/issues/183) I'm not planning to look for a new job for the time being.
